src/remediations/controller.write.js

```diff
@@ -170,7 +170,24 @@ exports.patch = errors.async(async function (req, res) {
         }
 
         if (name) {
+            const existing_remediation = await db.remediation.findOne({
+                attributes: ['id'],
+                where: { created_by: username, name: name },
+            }, {
+                transaction
+            });
+
+            // Do not allow users to use the same name for more than one remediation
+            if (existing_remediation) {
```

Comment:
What is the motivation to prevent duplicate names of playbooks? The ticket does not include any details on this.

Reply:
Here's the main ticket for this: https://issues.redhat.com/browse/RHINENG-4942

The 2 subtasks don't have much info but this explains it better.